tests/src/testEAS.hh

```diff
@@ -64,21 +66,23 @@ struct ElementTest<Ikarus::EnhancedAssumedStrains<DisplacementBasedElement>> {
         auto rule       = Dune::QuadratureRules<double, gridDim>::rule(localView.element().type(), order);
 
         if (numberOfEASParameter > 0) {
-          auto easVariantCopy    = fe.easVariant();  // This only test if the variant has a copy assignment operator
-          const auto& easVariant = fe.easVariant();
+          auto easVariantCopy    = fe.getEASVariant();  // This only test if the variant has a copy assignment operator
```

Comment:
can yout rename to the old function name `get...`is redundant I thnik and is also apropirate to the dune style of function naming

Reply:
done